.github/workflows/cron-job-its.yml

```diff
@@ -22,7 +22,8 @@ on:
       - 'owasp-dependency-check-suppressions.xml'
     branches:
       - master
-      - /^\d+\.\d+\.\d+(-\S*)?$/ # release branches
+      - '[0-9]+.[0-9]+.[0-9]+' # release branches
+      - '[0-9]+.[0-9]+.[0-9]+-[A-Za-z0-9]+' # release branches
```

Comment:
This is ok. But we could use [non-capturing groups](https://www.regular-expressions.info/brackets.html) to combine these two regexes for release branches into one:

```
[0-9]+.[0-9]+.[0-9]+(?:-\w+)?
```
Also, `[A-Za-z0-9]` -> `\w`